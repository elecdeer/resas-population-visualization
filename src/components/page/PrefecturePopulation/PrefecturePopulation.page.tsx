import React from "react";
import { PrefecturePopulation } from "./PrefecturePopulation";

export const PrefecturePopulationPage: React.VFC = () => {
  return <PrefecturePopulation />;
};
