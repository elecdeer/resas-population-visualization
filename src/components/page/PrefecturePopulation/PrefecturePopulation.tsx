import React from "react";
import styles from "./PrefecturePopulation.module.scss";

export type PrefecturePopulationProps = {
  
};

export const PrefecturePopulation: React.FC<PrefecturePopulationProps> = ({
  
}) => {
  return (
    <>
    </>
  );
};