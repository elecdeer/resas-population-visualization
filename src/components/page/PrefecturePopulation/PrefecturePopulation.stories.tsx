import { PrefecturePopulation, PrefecturePopulationProps} from "./PrefecturePopulation";

export default {
	title: "ui/PrefecturePopulation",
	component: PrefecturePopulation
}

export const Default = (args: PrefecturePopulationProps) => <PrefecturePopulation {...args}/>