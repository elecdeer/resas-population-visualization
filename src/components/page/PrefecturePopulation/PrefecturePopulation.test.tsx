import { PrefecturePopulation } from "./PrefecturePopulation";
import { render } from "@testing-library/react";

jest.mock("next/dist/client/router", () => ({
  __esModule: true,
  useRouter: () => ({
    query: {},
    pathname: "/",
    asPath: "/",
    events: {
      emit: jest.fn(),
      on: jest.fn(),
      off: jest.fn(),
    },
    push: jest.fn(() => Promise.resolve(true)),
    prefetch: jest.fn(() => Promise.resolve(true)),
    replace: jest.fn(() => Promise.resolve(true)),
  }),
}));

test("PrefecturePopulation Component Snapshot Test", () => {
  const { asFragment } = render(<PrefecturePopulation />);
  expect(asFragment()).toMatchSnapshot();
});
