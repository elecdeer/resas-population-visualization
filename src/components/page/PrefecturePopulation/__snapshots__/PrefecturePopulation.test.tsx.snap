// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrefecturePopulation Component Snapshot Test 1`] = `
<DocumentFragment>
  <p>
    Loading...
  </p>
</DocumentFragment>
`;
